tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2145,10 +2190,60 @@ def test_whisper_longform_single_batch(self):
 
         assert is_increasing
 
+    @slow
+    def test_whisper_longform_prompt_ids(self):
```

Comment:
This test shows the advantages of both `first-segment` and `all-segments` prompt ids for long-form generation.

Reply:
Very nice - this could go in our Whisper blog post / detailed long-form guide (or even the new lib)